watchfrr/watchfrr_vty.c

```diff
@@ -105,7 +105,10 @@ DEFUN(config_write_integrated,
 
 	/* don't allow the user to pass parameters, we're root here!
 	 * should probably harden vtysh at some point too... */
-	execl(VTYSH_BIN_PATH, "vtysh", "-w", NULL);
+	if (netns)
```

Comment:
It is allowed to use different `netns` and `pathspace` (`-N mypath --netns mynetns`), so we should use `pathspace` here, not netns name.

Reply:
Uh, I didn't _intend_ for that to work when I put the feature together, if that works it's purely accidental :)

Reply:
Or did I? I don't even remember...

Anyway, yes, `vtysh` needs to get a `-N pathspace` argument.